src/pages/gen2/build-a-backend/add-aws-services/custom-resources/index.mdx

```diff
@@ -18,7 +18,7 @@ Custom resources allow you to integrate any AWS service into an Amplify backend.
 
 </Callout>
 
-With Amplify (Gen 2), you can add custom AWS resources to an Amplify app using the [AWS Cloud Development Kit (AWS CDK)](https://docs.aws.amazon.com/cdk/latest/guide/home.html). The AWS CDK is an open source software development framework that defines your cloud application resources using familiar programming languages, such as TypeScript.
+With Amplify (Gen 2), you can add custom AWS resources to an Amplify app using the [AWS Cloud Development Kit (AWS CDK)](https://docs.aws.amazon.com/cdk/latest/guide/home.html), which is installed by default as part of the `create amplify` workflow. The AWS CDK is an open source software development framework that defines your cloud application resources using familiar programming languages, such as TypeScript.
```

Comment:
```suggestion
With Amplify (Gen 2), you can add custom AWS resources to an Amplify app using the [AWS Cloud Development Kit (AWS CDK)](https://docs.aws.amazon.com/cdk/latest/guide/home.html), which is installed by default as part of the `npm create amplify` workflow. The AWS CDK is an open source software development framework that defines your cloud application resources using familiar programming languages, such as TypeScript.
```

I think `create amplify` by itself can be confusing, but I'm a bit torn on whether we'd want the full command or the package name with a link

```md
[`create-amplify`](https://www.npmjs.com/package/create-amplify)
```